### PR TITLE
Added summary for Array.Fill<T>(T[], T)

### PR DESCRIPTION
**Pull request by FarhadShariatzadeh**
## Adding a short summary for Array.Fill<T>(T[], T) method

I just explained what this method does.

### CODE REVIEW

File: xml/System/Array.xml [hunk 1]
```diff
@@ -2769,7 +2769,7 @@
         <typeparam name="T">To be added.</typeparam>
```
Comment (rpetrusha) [inline xml/System/Array.xml]:
```suggestion
        <typeparam name="T">The type of the elements in the array.</typeparam>
```

File: xml/System/Array.xml [hunk 2]
```diff
@@ -2769,7 +2769,7 @@
...
         <param name="array">To be added.</param>
```
Comment (rpetrusha) [inline xml/System/Array.xml]:
```suggestion
        <param name="array">The array to be filled.</param>
```

File: xml/System/Array.xml [hunk 3]
```diff
@@ -2769,7 +2769,7 @@
...
         <param name="value">To be added.</param>
```
Comment (rpetrusha) [inline xml/System/Array.xml]:
```suggestion
        <param name="value">The value to assign to each array element.</param>
```

**Comment by FarhadShariatzadeh** [comment]
> Thank you, @FarhadShariatzadeh, for contributing to the dotnet/dotnet-api-docs repo and documenting one of the overloads of Array.Fill. I've made some suggestions so that the overload would be fully documented. Would you also like to document the Fill<T>(T[], T) overload?

My pleasure @rpetrusha. Thank you for reviewing my pull request. Yes, I like to document the Fill<T>(T[], T) overload as well. I will send a new pull request for that soon.

**Comment by FarhadShariatzadeh** [comment]
Hello @rpetrusha, I was wondering if my pull request is ready to merge. Would you please let me know if I need to make any more changes? Thank you!

**Comment by rpetrusha** [comment]
Yes, this is ready to merge, @FarhadShariatzadeh. Sorry for the delay. I've resolved a merge conflict, so I'll merge as soon as the build completes successfully.

**Comment by FarhadShariatzadeh** [comment]
> Yes, this is ready to merge, @FarhadShariatzadeh. Sorry for the delay. I've resolved a merge conflict, so I'll merge as soon as the build completes successfully.

Awesome! no problem, Thank you very much!!

**Pull request merged**